src/tools/processData.js

```diff
@@ -253,6 +256,7 @@ export const hideRecipientsGrants = async (recipientsGrants) => {
     );
     const trailingNumber = grant.id;
     const newGrantNumber = `0${faker.datatype.number({ min: 1, max: 9 })}${faker.animal.type()}0${trailingNumber}`;
```

Comment:
Are we sure this will produce enough variation to not have generated duplicates?

Reply:
9 x 14 = 126 max unique values generated by the faker as currently written.